airgun/views/contentviewfilter.py

```diff
@@ -0,0 +1,343 @@
+                    provided - all available errata in table will be selected
+                    (especially useful together with filtering)
+                :param dict optional filters: dictionary containing widget
+                    names and values to set (like with regular `fill()`)
+                """
+                self.search(errata_id, filters)
+                if errata_id:
+                    self.table.row((
+                        'Errata ID', errata_id))[0].widget.fill(True)
+                else:
+                    self.select_all.fill(True)
+                self.add_button.click()
+
+            def fill(self, errata_id=None, filters=None):
+                self.add(errata_id, filters)
+
+            def read(self):
+                return self.table.read()
+
+        @View.nested
+        class erratum_date_range(SatSecondaryTab):
+            TAB_NAME = 'Erratum Date Range'
+
+            security = Checkbox(locator=".//input[@ng-model='types.security']")
+            enhancement = Checkbox(
+                locator=".//input[@ng-model='types.enhancement']")
+            bugfix = Checkbox(
+                locator=".//input[@ng-model='types.bugfix']")
+            date_type = RadioGroup(
+                ".//div[label[contains(@class, 'radio-inline')]]")
+            start_date = DatePicker(
+                locator=".//input[@ng-model='rule.start_date']")
+            end_date = DatePicker(
+                locator=".//input[@ng-model='rule.end_date']")
+
+            save = Text('//button[contains(@ng-click, "handleSave()")]')
+            cancel = Text('//button[contains(@ng-click, "handleCancel()")]')
+
+            def after_fill(self, was_change):
+                self.save.click()
+
+        def add(self, errata_id=None, filters=None):
+            """Add specific errata to filter or all available if id not
+                provided.
+
+            :param str optional errata_id: ID of errata to add. If not
+                provided - all available errata in table will be selected
+                (especially useful together with filtering)
+            :param dict optional filters: dictionary containing widget names
+                and values to set (like with regular `fill()`)
+            """
+            return self.AddTab.fill(errata_id, filters)
+
+        def read(self):
+            """Read values from tabs depending on errata filter type (by id or
+            daterange filter).
+            """
+            if self.erratum_date_range.is_displayed:
+                return {'erratum_date_range': self.erratum_date_range.read()}
+            return {
+                'assigned': self.ListRemoveTab.read(),
+                'unassigned': self.AddTab.read(),
+            }
+
+    @content_tabs.register(
+        lambda filter_type: filter_type.endswith('Package Groups'))
+    class package_group_filter(AddRemoveResourcesView):
+        pass
```

Comment:
not implemented? if yes - add a comment about that

Reply:
Implemented, regular `AddRemoveResourcesView`, nothing special, so nothing new added.